### PR TITLE
feat: add categories & quizzes page

### PR DESCRIPTION
**Pull request by anik31**
Categories & Quizzes page for Quiz project.
category.html & quizzes.html contains the HTML code & style.css contains the CSS code.

Deploy link for categories - https://deploy-preview-5--vacay-quiz.netlify.app/category.html

Preview image for categories -
image

Deploy link for quizzes - https://deploy-preview-5--vacay-quiz.netlify.app/quizzes.html

Preview image for quizzes -
image

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -175,4 +179,51 @@ section{
...
+
+.category .img-round-sizing{
+    width: 18rem;
+    height: 18rem;
+}
+
+.category-container{
+    display: grid;
+    gap: 1rem;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
```
Comment (nayyyhaa) [inline style.css]: Clean code. Just one suggestion, you could use repeat syntax here for grid, rest nice work! 💯

**Review by nayyyhaa — APPROVED** [review]

File: style.css (hunk 1 above)
Comment (anik31) [reply style.css]: Thanks for the suggestion @nayyyhaa , I will do it.

File: category.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                <li><a href="#" class="btn btn-primary">Login</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main class="categories">
+        <h3>CATEGORIES</h3>
+        <div  class="category-container">
+            <a href="/quizzes.html" class="category">
+                <img class="img-round img-round-sizing" src="/assets/beach.jfif" alt="beach">
+                <text>Beaches</text>
```
Comment (ApurvaSawant11) [inline category.html:32]: Add some font-weight to the text. It's a bit difficult to read depending on the background.

**Review by ApurvaSawant11 — APPROVED** [review]
Everything looks good!

File: category.html (hunk 2 above)
Comment (anik31) [reply category.html:32]: Okay I will do this

**Pull request merged**